### DISCUSSION TITLE
Adding a way to check stored enchantments in enchanted book

### DISCUSSION THREAD

**Discussion by lofez0109** [SkriptLang/Skript#7249] [Suggestions]
My idea is that a way to check the stored enchantments in enchanted book should be added. For example, if I try to get stored enchantments of mending enchanted book, 'Mending' should be value of it. I saw some discussion about this topic 3-4 years ago, but any other way to do this was suggested or added except for the way using skript-mirror or reflect. If expression to check the value of stored enchantments is added, it could be like :

`set {_a} to stored enchantments of {_b}`

({_b} is enchanted book of something)

I'm sorry if the way without sk-mirror/reflect that I couldn't find was existed, but I hope it will be added someday.

**Comment by AyhamAl-Ali** [comment]
There is a PR (#4366) adding this feature. It's been stale for sometime, hopefully someone take over it.
